### ISSUE TITLE
Missing metrics files

### ISSUE THREAD

**Issue by milleruntime** [apache/accumulo#1663]
I was running CI on the 1.9 branch and saw these warnings printed at startup of the tserver.
```
2020-07-28 17:50:32,495 [impl.MetricsSystemImpl] WARN : Error creating sink 'file-gc'
org.apache.hadoop.metrics2.impl.MetricsConfigException: Error creating plugin: org.apache.hadoop.metrics2.sink.FileSink
        at org.apache.hadoop.metrics2.impl.MetricsConfig.getPlugin(MetricsConfig.java:206)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.newSink(MetricsSystemImpl.java:532)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.configureSinks(MetricsSystemImpl.java:503)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.configure(MetricsSystemImpl.java:479)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.start(MetricsSystemImpl.java:188)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.init(MetricsSystemImpl.java:163)
        at org.apache.hadoop.metrics2.lib.DefaultMetricsSystem.init(DefaultMetricsSystem.java:54)
        at org.apache.hadoop.metrics2.lib.DefaultMetricsSystem.initialize(DefaultMetricsSystem.java:50)
        at org.apache.accumulo.server.metrics.MetricsSystemHelper$MetricsSystemHolder.<clinit>(MetricsSystemHelper.java:47)
        at org.apache.accumulo.server.metrics.MetricsSystemHelper.getInstance(MetricsSystemHelper.java:51)
        at org.apache.accumulo.tserver.metrics.TabletServerMetricsFactory.<init>(TabletServerMetricsFactory.java:46)
        at org.apache.accumulo.tserver.TabletServer.<init>(TabletServer.java:450)
        at org.apache.accumulo.tserver.TabletServer.main(TabletServer.java:3242)
        at org.apache.accumulo.tserver.TServerExecutable.execute(TServerExecutable.java:33)
        at org.apache.accumulo.start.Main$1.run(Main.java:130)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.hadoop.metrics2.MetricsException: Error creating ./target/accgc.metrics
        at org.apache.hadoop.metrics2.sink.FileSink.init(FileSink.java:53)
        at org.apache.hadoop.metrics2.impl.MetricsConfig.getPlugin(MetricsConfig.java:202)
        ... 15 more
Caused by: java.io.FileNotFoundException: ./target/accgc.metrics (No such file or directory)
        at java.io.FileOutputStream.open0(Native Method)
        at java.io.FileOutputStream.open(FileOutputStream.java:270)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:162)
        at org.apache.hadoop.metrics2.sink.FileSink.init(FileSink.java:49)
        ... 16 more
```
```
2020-07-28 17:50:32,500 [impl.MetricsSystemImpl] WARN : Error creating sink 'file-all'
org.apache.hadoop.metrics2.impl.MetricsConfigException: Error creating plugin: org.apache.hadoop.metrics2.sink.FileSink
        at org.apache.hadoop.metrics2.impl.MetricsConfig.getPlugin(MetricsConfig.java:206)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.newSink(MetricsSystemImpl.java:532)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.configureSinks(MetricsSystemImpl.java:503)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.configure(MetricsSystemImpl.java:479)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.start(MetricsSystemImpl.java:188)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.init(MetricsSystemImpl.java:163)
        at org.apache.hadoop.metrics2.lib.DefaultMetricsSystem.init(DefaultMetricsSystem.java:54)
        at org.apache.hadoop.metrics2.lib.DefaultMetricsSystem.initialize(DefaultMetricsSystem.java:50)
        at org.apache.accumulo.server.metrics.MetricsSystemHelper$MetricsSystemHolder.<clinit>(MetricsSystemHelper.java:47)
        at org.apache.accumulo.server.metrics.MetricsSystemHelper.getInstance(MetricsSystemHelper.java:51)
        at org.apache.accumulo.tserver.metrics.TabletServerMetricsFactory.<init>(TabletServerMetricsFactory.java:46)
        at org.apache.accumulo.tserver.TabletServer.<init>(TabletServer.java:450)
        at org.apache.accumulo.tserver.TabletServer.main(TabletServer.java:3242)
        at org.apache.accumulo.tserver.TServerExecutable.execute(TServerExecutable.java:33)
        at org.apache.accumulo.start.Main$1.run(Main.java:130)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.hadoop.metrics2.MetricsException: Error creating ./target/it.all.metrics
        at org.apache.hadoop.metrics2.sink.FileSink.init(FileSink.java:53)
        at org.apache.hadoop.metrics2.impl.MetricsConfig.getPlugin(MetricsConfig.java:202)
        ... 15 more
Caused by: java.io.FileNotFoundException: ./target/it.all.metrics (No such file or directory)
        at java.io.FileOutputStream.open0(Native Method)
        at java.io.FileOutputStream.open(FileOutputStream.java:270)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:162)
        at org.apache.hadoop.metrics2.sink.FileSink.init(FileSink.java:49)
        ... 16 more
```
```
2020-07-28 17:50:32,501 [impl.MetricsSystemImpl] WARN : Error creating sink 'file-master'
org.apache.hadoop.metrics2.impl.MetricsConfigException: Error creating plugin: org.apache.hadoop.metrics2.sink.FileSink
        at org.apache.hadoop.metrics2.impl.MetricsConfig.getPlugin(MetricsConfig.java:206)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.newSink(MetricsSystemImpl.java:532)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.configureSinks(MetricsSystemImpl.java:503)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.configure(MetricsSystemImpl.java:479)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.start(MetricsSystemImpl.java:188)
        at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.init(MetricsSystemImpl.java:163)
        at org.apache.hadoop.metrics2.lib.DefaultMetricsSystem.init(DefaultMetricsSystem.java:54)
        at org.apache.hadoop.metrics2.lib.DefaultMetricsSystem.initialize(DefaultMetricsSystem.java:50)
        at org.apache.accumulo.server.metrics.MetricsSystemHelper$MetricsSystemHolder.<clinit>(MetricsSystemHelper.java:47)
        at org.apache.accumulo.server.metrics.MetricsSystemHelper.getInstance(MetricsSystemHelper.java:51)
        at org.apache.accumulo.tserver.metrics.TabletServerMetricsFactory.<init>(TabletServerMetricsFactory.java:46)
        at org.apache.accumulo.tserver.TabletServer.<init>(TabletServer.java:450)
        at org.apache.accumulo.tserver.TabletServer.main(TabletServer.java:3242)
        at org.apache.accumulo.tserver.TServerExecutable.execute(TServerExecutable.java:33)
        at org.apache.accumulo.start.Main$1.run(Main.java:130)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.hadoop.metrics2.MetricsException: Error creating ./target/master.metrics
        at org.apache.hadoop.metrics2.sink.FileSink.init(FileSink.java:53)
        at org.apache.hadoop.metrics2.impl.MetricsConfig.getPlugin(MetricsConfig.java:202)
        ... 15 more
Caused by: java.io.FileNotFoundException: ./target/master.metrics (No such file or directory)
        at java.io.FileOutputStream.open0(Native Method)
        at java.io.FileOutputStream.open(FileOutputStream.java:270)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:162)
        at org.apache.hadoop.metrics2.sink.FileSink.init(FileSink.java:49)
        ... 16 more
```

**Comment by EdColeman** [comment]
That's because the config files are setup for testing with maven - the "fix" would be to modify the files to point to a real directory on the test system.  Probably need to see about adding sensible defaults to the assembly so that they get included with the tarball and the other sample configs.

**Comment by milleruntime** [comment]
> Probably need to see about adding sensible defaults to the assembly so that they get included with the tarball and the other sample configs.

Yeah I just thought we should do something so we don't see these in the release candidate.

**Comment by EdColeman** [comment]
It looks likes the test properties file is being bundled into the accumulo-test.jar.  The prop files in assemble/templates look okay, but they should be updated with the "new" file possibilities and commented out like the rest so that there are samples for people to uncomment out.  Working on it now.

**Comment by EdColeman** [comment]
Closed with PR #1663

**Comment by EdColeman** [comment]
Closed with PR #1663
